runbooks/key-handover/key-handover.md

```diff
@@ -0,0 +1,252 @@
+
+> [!NOTE] 
+> The mainnet address is a [2-of-2 multisig](https://etherscan.io/address/0x5a0Aae59D09fccBdDb6C6CcEB07B7279367C3d2A#readProxyContract). One owner is an Optimism Foundation 5/7 [multisig](https://etherscan.io/address/0x847B5c174615B1B7fDF770882256e2D3E95b9D92#readProxyContract) and the other owner is the [Security Council's](https://gov.optimism.io/t/intro-to-optimisms-security-council/6885) [multisig](https://etherscan.io/address/0xc2819DC788505Aac350142A7A707BF9D03E3Bd03#readProxyContract).
+
+> [!IMPORTANT]
+> The prerequisite for the Key Handover upgrade is that the chain must be a Standard Chain and passes the corresponding [block history integrity checks](https://github.com/ethereum-optimism/security-tools/tree/main/src/block-history-integrity-checks). 
+
+## Upgrade Process
+
+### Setup
+
+#### **Local Machine**
+
+First, let’s make sure you have all the right repos and tools on your machine. Start by cloning the repos below and checking out the latest main branch unless stated otherwise. Then, follow the repo setup instructions for each.
+
+1. https://github.com/ethereum-optimism/superchain-ops
+    1. Follow the installation instructions in the README: https://github.com/ethereum-optimism/superchain-ops?tab=readme-ov-file#installation
+    2. Then, run `just install`.
+2. https://github.com/ethereum-optimism/superchain-registry
+    1. No setup steps.
+3. Ensure you have a Tenderly account.
+
+#### Familiarize yourself with the `single.just` file (superchain-ops repo)
+
+There are three just recipes in this file:
+
+- `simulate` - to simulate the transactions in the the `input.json` bundle
+- `sign` - to sign the transactions in the `input.json` bundle
+- `execute` - to execute the transactions in the `input.json` bundle
+
+We use `single.just` because the ProxyAdmin owners are regular Safe’s. (For OP Mainnet we use `nested.just` because the ProxyAdmin owner is a Safe, where both owners are also Safes).
+
+### Scaffold the ops task (playbook) for your upgrade (superchain-ops repo)
+
+#### Create a task directory in superchain-ops
+
+```bash
+mkdir tasks/<NETWORK_DIR>/<RUNBOOK_DIR>
+```
+
+In the superchain-ops repo, tasks live in `tasks/<NETWORK_DIR>/<RUNBOOK_DIR>` where:
+
+- `NETWORK_DIR` is `eth` for Ethereum mainnet and `sep` for Sepolia.
+- `RUNBOOK_DIR` is of the form `{chainName}-{upgradeIndex}-{upgradeName}`.
+    - `chainName` is just the chain’s name i.e. `base` . This is excluded for OP Chains.
+    - `upgradeIndex` starts at `001` for the first playbook and increments each time. This gives a sequential ordering to upgrade transactions occurring on that chain.
+    - `upgradeName` is `key-handover`
+
+#### Copy the following files into the task directory
```

Comment:
I see why these files are in this directory. I think that you can just reference the corresponding mmz files and make a point of saying that they should modify for their networks.